package.json

```diff
@@ -30,7 +30,7 @@
     "require": "test/setup-env.js"
   },
   "peerDependencies": {
-    "@adobe/helix-universal": ">=1.6.0"
+    "@adobe/helix-universal": "adobe/helix-universal#edge-deploy"
```

Comment:
we should not merge until this is released.

Reply:
I don't use the handlers in this branch, instead there is a new entry point here: https://github.com/adobe/helix-deploy/pull/324/files#diff-1acdf076388cf828a0d46abff430138d020e5933f7e69ff29819eb462857980fR58

Adding this into `helix-universal` is not a good idea, because we'd have to either bundle the AWS/Azure/Google SDK and hit Cloudflare's size limit, or we have to live with some dependencies being unsatisfied, which the Cloudflare runtime also does not like. Instead I'd wait until the service worker wrappers are halfway stable, then refactor them out into a separate `helix-universal-edge` package.

Reply:
but we cannot have a peer dependency to a github package. otherwise all services need to update their helix-universal dependency too...

Reply:
Got it. I reverted this change.